### ISSUE TITLE
Can't compile properly after migrating to higher version(s)

### ISSUE THREAD

**Issue by paul-buechner** [gquinteros93/angular-css-modules#4]
Hi German,

## Migration steps
So basically after starting migrating to the next **major** version which was done by:
`ng update @angular/cli@^9 @angular/core@^9`

## Error serving with `ng serve`
You start to get the following error message by serving the app with `ng serve`:
`An unhandled exception occurred: Cannot read property 'options' of undefined`

![grafik](https://user-images.githubusercontent.com/45827409/105091955-e51bbe80-5aa0-11eb-9f27-deb1b65b1f75.png)

and the following error-log:
```

[error] TypeError: Cannot read property 'options' of undefined
at module.exports (C:\...\angular-css-modules\extra-webpack.config.js:11:38)
at Function.buildWebpackConfig (C:\...\angular-css-modules\node_modules\@angular-builders\custom-webpack\dist\custom-webpack-builder.js:15:20)
at Object.webpackConfiguration (C:\...\angular-css-modules\node_modules\@angular-builders\custom-webpack\dist\common.js:14:58)
at setup (C:\...\angular-css-modules\node_modules\@angular-devkit\build-angular\src\dev-server\index.js:115:46)
at processTicksAndRejections (internal/process/task_queues.js:93:5)

```

## Error serving with `npm run start:app`
Serving the app with `npm run start:app` gives some different error loggs:
```

An unhandled exception occurred: Cannot read property 'options' of undefined
See "C:\Users\Admin\AppData\Local\Temp\ng-53Ung4\angular-errors.log" for further details.
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file C:\WINDOWS\system32\cmd.exe
npm ERR! errno ENOENT
npm ERR! angular-css-modules@0.0.0 start:app: `ng serve angular-css-modules`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the angular-css-modules@0.0.0 start:app script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
and ...

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'start:app'
1 verbose cli ]
2 info using npm@6.14.7
3 info using node@v14.8.0
4 verbose run-script [ 'prestart:app', 'start:app', 'poststart:app' ]
5 info lifecycle angular-css-modules@0.0.0~prestart:app: angular-css-modules@0.0.0
6 info lifecycle angular-css-modules@0.0.0~start:app: angular-css-modules@0.0.0
7 verbose lifecycle angular-css-modules@0.0.0~start:app: unsafe-perm in lifecycle true
8 verbose lifecycle angular-css-modules@0.0.0~start:app: PATH: C:\Program Files\...
9 verbose lifecycle angular-css-modules@0.0.0~start:app: CWD: C:\...\angular-css-modules
10 silly lifecycle angular-css-modules@0.0.0~start:app: Args: [ '/d /s /c', 'ng serve angular-css-modules' ]
11 info lifecycle angular-css-modules@0.0.0~start:app: Failed to exec start:app script
12 verbose stack Error: angular-css-modules@0.0.0 start:app: `ng serve angular-css-modules`
12 verbose stack spawn ENOENT
12 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:48:18)
12 verbose stack     at ChildProcess.emit (events.js:314:20)
12 verbose stack     at maybeClose (internal/child_process.js:1051:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
13 verbose pkgid angular-css-modules@0.0.0
14 verbose cwd C:\...\angular-css-modules
15 verbose Windows_NT 10.0.19042
16 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "start:app"
17 verbose node v14.8.0
18 verbose npm  v6.14.7
19 error code ELIFECYCLE
20 error syscall spawn
21 error file C:\WINDOWS\system32\cmd.exe
22 error errno ENOENT
23 error angular-css-modules@0.0.0 start:app: `ng serve angular-css-modules`
23 error spawn ENOENT
24 error Failed at the angular-css-modules@0.0.0 start:app script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 1, true ]
```

## Reasoning/Fixing
I tried fixing it by updating angular-builder to version 9 but nothing happened. Maybe there is some **new syntax interpretation** regarding the **@angular-builders**-compiler or **webpack.config**...?

**Comment by paul-buechner** [comment]
Here is the `package.json` to reproduce:

```
"dependencies": {
+-- @angular/animations@9.1.13
+-- @angular/cdk@8.2.3
+-- @angular/common@9.1.13
+-- @angular/compiler@9.1.13
+-- @angular/core@9.1.13
+-- @angular/forms@9.1.13
+-- @angular/material@8.2.3
+-- @angular/platform-browser@9.1.13
+-- @angular/platform-browser-dynamic@9.1.13
+-- @angular/router@9.1.13
+-- body-parser@1.19.0
+-- express@4.17.1
+-- hammerjs@2.0.8
+-- rxjs@6.6.3
+-- tslib@1.11.1
`-- zone.js@0.10.3
},
 "devDependencies": {
+-- @angular-builders/custom-webpack@9.2.0
+-- @angular-devkit/build-angular@0.901.13
+-- @angular/cli@9.1.13
+-- @angular/compiler-cli@9.1.13
+-- @angular/language-service@9.1.13
+-- @types/jasmine@3.3.16
+-- @types/jasminewd2@2.0.8
+-- @types/node@12.19.14
+-- codelyzer@5.2.2
+-- jasmine-core@3.6.0
+-- jasmine-spec-reporter@4.2.1
+-- karma@4.1.0
+-- karma-chrome-launcher@2.2.0
+-- karma-coverage-istanbul-reporter@2.0.6
+-- karma-jasmine@2.0.1
+-- karma-jasmine-html-reporter@1.5.3
+-- lodash@4.17.19
+-- postcss-loader@3.0.0
+-- postcss-modules@1.5.0
+-- posthtml-css-modules@0.1.3
+-- posthtml-loader@1.0.2
+-- protractor@5.4.3
+-- raw-loader@3.1.0
+-- ts-node@7.0.1
+-- tslint@5.15.0
+-- typescript@3.8.3
`-- webpack@4.42.1
}
```

**Comment by gquinteros93** [comment]
Hi @paul-buechner, thank you for reporting the error and the detailed information. 

You were right, the error was happening. 

After some research, I founded the issue. 

The issue was In the `line 10` of the `extra-webpack.config.js` file:

```
const postcssLoader = scssRule.use.find(x => x.loader === 'postcss-loader');
```

After changing that line for: 

```
const postcssLoader = scssRule.use.find(x => x.loader.toString().includes('postcss-loader'));
```

Everything is working as expected and how it was working in v8.

Please, let me know if that fixes the issue you were having too.

Thanks.

**Comment by paul-buechner** [comment]
Fixed issue #4

1bdd77ba9ba73e377113329c9f8f2e6ecf8bb030
